aptos-move/framework/aptos-stdlib/sources/cryptography/algebra.move

```diff
@@ -1,13 +1,42 @@
 /// This module provides generic structs/functions for operations of algebraic structures (e.g. fields and groups),
 /// which can be used to build generic cryptographic schemes atop.
-/// See `algebra_*.move` for currently implemented algebraic structures.
 ///
-/// Currently supported operations include element serialization/deserialization and addition.
+/// In general, every structure implements basic operations like (de)serialization, equality check, random sampling.
+///
+/// A group may also implement the following operations. (Additive notions are used.)
```

Comment:
Replace what's in the parentheses `()` with: (Additive group notation is assumed.)

Reply:
Fixed.